### ISSUE TITLE
The experimental results of TResnet_M cannot be reproduced

### ISSUE THREAD

**Issue by limengyang1992** [Alibaba-MIIL/ML_Decoder#50]
Hello, I used the pre-training model of TResnet_M, the image input size is 224, the learning rate is 2e-4 in the paper, the rest of the parameters are consistent with the training file, I only got 78.86 results instead of 84.2 in MODEL_ZERO.Do I need to make any changes?

**Comment by limengyang1992** [comment]
My pre-training weights come from TResNet pre-trained models
